decisiontree/models.py

```diff
@@ -198,6 +199,27 @@ def __str__(self):
         return u"%s : %s --> %s" % (self.current_state, self.answer, self.next_state)
 
 
+class SessionQuerySet(models.query.QuerySet):
+
+    def open(self):
+        return self.exclude(Q(state=None) | Q(canceled=True))
```

Comment:
Nope, no good reason to avoid them. For some reason, my brain reads the non-Q version easier, but I agree with your point about consistency. I like the refactored version even more.